CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [12-2.2] - 2021-09-29
+* Initial release for Android 12
```

Comment:
The initial release for Android 12 will hopefully be `12-3.0`

Reply:
Alright let's just leave this one open for now then.